contracts/pcv/PCVDepositAggregator.sol

```diff
@@ -0,0 +1,444 @@
+    function balance() public view override returns (uint256) {
+        return IERC20(token).balanceOf(address(this));
+    }
+
+    /// @notice current percent of PCV held by the input `pcvDeposit` relative to the total managed by aggregator.
+    /// @param pcvDeposit the address of the pcvDeposit
+    /// @param depositAmount a hypothetical deposit amount, to be included in the calculation
+    /// @return the percent held as a Decimal D256 value
+    function percentHeld(address pcvDeposit, uint256 depositAmount) external view override returns(Decimal.D256 memory) {
+        uint256 totalBalanceWithTheoreticalDeposit = getTotalBalance() + depositAmount;
+        uint256 targetBalanceWithTheoreticalDeposit = IPCVDeposit(pcvDeposit).balance() + depositAmount;
+
+        return Decimal.ratio(targetBalanceWithTheoreticalDeposit, totalBalanceWithTheoreticalDeposit);
+    }
+
+    /// @notice the normalized target weight of PCV held by `pcvDeposit` relative to aggregator total
+    /// @param pcvDeposit the address of the pcvDeposit
+    /// @return the normalized target percent held as a Decimal D256 value
+    function normalizedTargetWeight(address pcvDeposit) public view override returns(Decimal.D256 memory) {
+        return Decimal.ratio(pcvDepositWeights[pcvDeposit], totalWeight);
+    }
+
+    /// @notice the raw amount of PCV off of the target weight/percent held by `pcvDeposit`
+    /// @dev a positive result means the target has "too much" pcv, and a negative result means it needs more pcv
+    /// @param pcvDeposit the address of the pcvDeposit
+    /// @return the amount from target as an int
+    function amountFromTarget(address pcvDeposit) public view override returns(int256) {
+        uint256 totalBalance = getTotalBalance();
+
+        uint256 pcvDepositBalance = IPCVDeposit(pcvDeposit).balance();
+        uint256 pcvDepositWeight = pcvDepositWeights[address(pcvDeposit)];
+
+        uint256 idealDepositBalance = pcvDepositWeight * totalBalance / totalWeight;
+        
+        return (pcvDepositBalance).toInt256() - (idealDepositBalance).toInt256();
+    }
+
+    /// @notice the same as amountFromTarget, but for every targets
+    /// @return distancesToTargets all amounts from targets as a uint256 array
+    function getAllAmountsFromTargets() public view override returns(int256[] memory distancesToTargets) {
+        (uint256 aggregatorBalance, uint256 underlyingSum, uint256[] memory underlyingBalances) = _getUnderlyingBalancesAndSum();
+        uint256 totalBalance = aggregatorBalance + underlyingSum;
+
+        distancesToTargets = new int256[](pcvDepositAddresses.length());
+
+        for (uint256 i=0; i < distancesToTargets.length; i++) {
+            uint256 idealAmount = totalBalance * pcvDepositWeights[pcvDepositAddresses.at(i)] / totalWeight;
+            distancesToTargets[i] = (idealAmount).toInt256() - (underlyingBalances[i]).toInt256();
+        }
+
+        return distancesToTargets;
```

Comment:
Do you imagine either of these methods being useful and worth keeping? They don't share code anywhere with the above. I had originally thought they might be useful but now I'm not sure

Reply:
could be useful for stats and stuff. doesn't hurt to keep it in.